src/config/http.rst

```diff
@@ -24,10 +24,10 @@ HTTP Server Options
 
 .. config:section:: chttpd :: Clustered HTTP Server Options
 
-.. note::
-    In CouchDB 2.x and 3.x, the `chttpd` section refers to the standard, clustered
-    port. All use of CouchDB, aside from a few specific maintenance tasks as
-    described in this documentation, should be performed over this port.
+    .. note::
```

Comment:
Was this whitespace change deliberate?

Reply:
To separate the `Note` msg from configuration options, I added indentation before these lines.
I separate to another commit 12289387e70be4311dd4721eeaf7c7006bc3f210 for this change.
Thanks for the review, @jaydoane 
![image](https://user-images.githubusercontent.com/54631519/120551463-ad604400-c3bb-11eb-8ef7-ed2c9ff7a467.png)